### ISSUE TITLE
Not able to shutdown notebooks (jupyter notebook version 4.0.6)

### ISSUE THREAD

**Issue by pybokeh** [jupyter/notebook#523]
My terminal log outputs this message when I click on the "Shutdown" button in the "Running" tab:
404 DELETE /jupyter/api/sessions/%5Bobject%20Object%5D (127.0.0.1) 55.14ms referer=http://localhost:8989/jupyter/tree

On Xubuntu 14.04 machine using Firefox. At the moment, I don't have another browser installed.

jupyter notebook installed via pip with conda environment activated.

**Comment by ellisonbg** [comment]
We need to try and replicate this...

**Comment by pybokeh** [comment]
Not positive but I think it is related to the fact that I installed jupyter
with pip and not conda.  I am using conda but installed jupyter with pip
because I was having trouble getting notebook entensions to work with the
version that was in conda distribution.  I guess my extensions problem was
fixed when jupyter was upgraded to 4.0.6., but then realized I have this
problem with not being able to shutdown notebooks.  Then I wondered if
jupyter got updated in conda so I re-made my conda environment and
installed jupyter with conda instead of pip.  Now, everything's good!
